src/app/util/attribute-storage.cpp

```diff
@@ -699,25 +699,12 @@ uint8_t emberAfClusterIndex(EndpointId endpoint, ClusterId clusterId, EmberAfClu
     return 0xFF;
 }
 
-// Returns whether the given endpoint has the client or server of the given
-// cluster on it.
-bool emberAfContainsCluster(EndpointId endpoint, ClusterId clusterId)
-{
-    return (emberAfFindCluster(endpoint, clusterId, 0) != nullptr);
-}
-
 // Returns whether the given endpoint has the server of the given cluster on it.
 bool emberAfContainsServer(EndpointId endpoint, ClusterId clusterId)
 {
     return (emberAfFindCluster(endpoint, clusterId, CLUSTER_MASK_SERVER) != nullptr);
 }
 
-// Returns whether the given endpoint has the client of the given cluster on it.
-bool emberAfContainsClient(EndpointId endpoint, ClusterId clusterId)
-{
-    return (emberAfFindCluster(endpoint, clusterId, CLUSTER_MASK_CLIENT) != nullptr);
```

Comment:
Followup: I think after this all non-dead call-sites of `emberAfFindCluster` pass `CLUSTER_MASK_SERVER`.  In particular, I think `emberAfClusterDefaultResponseCallback` and `emberAfClusterMessageSentCallback` are dead code and should be removed.

Once that happens, we should just remove that argument and rename the function to `emberAfFindServerCluster`...

Reply:
I will do the removal of CLUSTER_MASK_SERVER as a followup. For now just deleting unused code.

Did remove the emberAfClusterDefaultResponseCallback and emberAfClusterMessageSentCallback